pages/mi/profiles/rlam20.md

```diff
@@ -0,0 +1,37 @@
+# Rayyan Alam
+
+## Information
+
+| Name | Timezone | Location | Operating System     |
+|-----------------------|---------------------|----------|----------|
+| Rayyan Alam   | EST | Virginia, United States   | Windows 10.0.22631|
+
+
+Hello! My name is Rayyan, and I'm an aspiring software developer. I'm passionate about computer science and electrical engineering
+
+## Skills
+
+- **Programming Languages:**  Python, Java, HTML/CSS
+- **Frameworks/Libraries:**   React.js, Bootstrap
+- **Tools:**   Git, VS Code, Android Studio
+  
+## Interests
+
+- Photography 📸
+- Basketball 🏀
+- Biking 🚴
+
+## Goals for the summer
+- [ ] Traveling
+- [x] Learn a new language
+- [ ] Calisthenics 
```

Comment:
maybe something like below would look even better ...
- Traveling
- Learn a new language ✔️
- Calisthenics 

... anyhow good job

Reply:
looks our old mdwiki is not able to show this md element properly
<img width="275" alt="image" src="https://github.com/open-learning-exchange/open-learning-exchange.github.io/assets/13684059/1b0fc2f9-ae8f-4684-814b-f7b4d2c6fce0">

Reply:
@rlam20 I still like your solution the best ... though the combo of strikethrough and checkmark is growing on me